### DISCUSSION TITLE
when is the next pre-release coming out on flathub?

### DISCUSSION THREAD

**Discussion by RustoMCSpit** [SuperTux/supertux#2806] [General]
hi all! the last flathub release is from years ago so just wondering when that will get an update

**Comment by ghnp5** [comment]
0.7.0 is released, and the `.flatpak` file can be downloaded in the Release.

However, I'm wondering when will the version in Flathub be updated.

From what I'm reading, the Flathub app isn't official.
But I cannot find any other official entry in the Flathub.
